python/cli/prelude_cli/views/scm.py

```diff
@@ -14,7 +14,8 @@ def scm(ctx):
 
 @scm.command('endpoints')
 @click.option('--partner',
-              type=click.Choice([c.name for c in Control if c != Control.INVALID], case_sensitive=False), default=None)
+              type=click.Choice([c.name for c in Control if c.category in [ControlCategory.XDR, ControlCategory.ASSET_MANAGER]], case_sensitive=False),
```

Comment:
is this just to make the cli easy to use? since endpoints doesn't take partner anymore?

Reply:
ehm. no. this was me forgetting to update the CLI 👍